pandas/core/tools/timedeltas.py

```diff
@@ -18,25 +18,36 @@
 
 def to_timedelta(arg, unit='ns', box=True, errors='raise'):
     """
-    Convert argument to timedelta
+    Convert argument to timedelta.
+
+    Timedeltas are absolute differences in times, expressed in difference
+    units e.g. (days, hours, minutes, seconds). This method converts an argument
+    from a recognized timedelta format / value into a Timedelta type.
 
     Parameters
     ----------
-    arg : string, timedelta, list, tuple, 1-d array, or Series
-    unit : unit of the arg (D,h,m,s,ms,us,ns) denote the unit, which is an
-        integer/float number
-    box : boolean, default True
-        - If True returns a Timedelta/TimedeltaIndex of the results
-        - if False returns a np.timedelta64 or ndarray of values of dtype
-          timedelta64[ns]
+    arg : str, timedelta, list, tuple, 1-d array, or Series
```

Comment:
I'd replace list, tuple, 1-d array by `list-like`. I'd also remove the comma vefore the or.